src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -31,6 +31,55 @@ def get_location_completion_list(cmd, prefix, namespace, **kwargs):  # pylint: d
     return [l.name for l in result]
 
 
+# pylint: disable=redefined-builtin
+def get_datetime_type(help=None, date=True, time=True, timezone=True):
+
+    help_string = help + ' ' or ''
```

Comment:
`or` does have higher precedence so the code is good. But I guess for the readability this form is more clear: `(help + '  ') or ''`